src/main/java/seedu/academydirectory/logic/commands/SortCommand.java

```diff
@@ -74,8 +73,8 @@ public String getResultString() {
     @Override
     public CommandResult execute(VersionedModel model) {
         requireNonNull(model);
-        ObservableList<Student> students = model.getFilteredStudentList();
 
+        List<Student> students = model.getAcademyDirectory().getStudentList();
         Comparator<Student> comparator = getComparator();
 
```

Comment:
This won't affect anything right? Considering that we're using ObservableList for FX.

Otherwise I think it's fine already.

Reply:
Nope! I refresh the View to view all students at the end of the command.